integrations.json

```diff
@@ -105,5 +105,58 @@
         }
       }
     ]
+  },
+  {
+    "name": "StackExchangeRedis",
+    "method_replacements": [
+      {
+        "caller": {},
+        "target": {
+          "assembly": "StackExchange.Redis",
+          "type": "StackExchange.Redis.ConnectionMultiplexer",
+          "method": "Connect",
+          "signature_types": [
+            "StackExchange.Redis.ConnectionMultiplexer",
+            "StackExchange.Redis.ConfigurationOptions",
+            "System.IO.TextWriter"
+          ],
+          "minimum_major": 2,
+          "minimum_minor": 0,
+          "minimum_patch": 0,
```

Comment:
Do we know if this could actually be set to the minimum version that we care about? Does the native code see a different version number than the managed code?

Reply:
As I understand current implementation, we are using `AssemblyVersion` to define ranges.
I do not see any option to narrow this versions with current implementation. We should be able to add validation on another attribute.